tests/metadata/card_metadata_integration_test.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from checkout_sdk.common.enums import CardMetadataFormatType
+from checkout_sdk.metadata.metadata import CardMetadataRequest, CardMetadataBinSource
+from tests.checkout_test_utils import assert_response
+
+
+def test_should_request_card_metadata(oauth_api):
```

Comment:
Also, isn't the other asserts missing? Card,Token?

Reply:
👍